charts/tekton-pipelines/values.yaml

```diff
@@ -351,6 +358,47 @@ buildpacks:
 kaniko:
   # -- should we enable the kaniko pipeline
   enabled: false
+  # needs to build static to be able to push it in Sentry during deploy
+  # there are no other ways to do this, because kaniko builder builds frontend
+  # deps right inside of the docker image (so we can't take it from it)
+  buildTaskSteps:
+    - name: build-static
+      image: node:$(params.node_version)
+      imagePullPolicy: IfNotPresent
+      resources: {}
+      workingDir: $(resources.inputs.app.path)
+      script: |
+        #!/usr/bin/env bash
+        set -o pipefail
+
+        if [[ ! -f package.json ]]; then
+          echo "No package.json found, stepping out"
+        else
+          echo "Installing node.js dependencies"
+          # set cache
+          npm config set cache $(workspaces.source.path)/.npm --global
+          npm ci
+          if [[ $? -ne 0 ]]; then
+            echo "unable to install dependencies, exit_code: $?"
+            exit 1
+          fi
+          echo "Building $(params.environment) target"
+          npm run build:$(params.environment)
+          if [[ $? -ne 0 ]];  then
+            echo "unable to build $(params.environment) exit_code: $?"
+            exit 1
+          fi
+          echo "Cleaning up"
+          rm -rf node_modules
+
+          # copy built dir to workspace for further usage in sentry-release task (source maps release)
```

Comment:
same question - where is the code to generate the source maps?

Reply:
they are generated during `npm run build: dev` command

Reply:
@dmitry-mightydevops removed this (described why in PR description in `kaniko-frontend` section)